circus-train-core/src/main/java/com/hotels/bdp/circustrain/core/conf/CircusTrainTransformOptions.java

```diff
@@ -15,26 +15,20 @@
  */
 package com.hotels.bdp.circustrain.core.conf;
 
-import static com.hotels.bdp.circustrain.core.conf.TableParametersConfig.TABLE_REPLICATION_TABLE_PARAMETERS;
-
-import java.util.Map;
+import static com.hotels.bdp.circustrain.core.conf.CircusTrainTransformOptions.TRANSFORM_OPTIONS_BEAN;
+import static com.hotels.bdp.circustrain.core.conf.CircusTrainTransformOptions.TRANSFORM_OPTIONS_PROPERTY;
 
 import org.springframework.boot.context.properties.ConfigurationProperties;
 import org.springframework.context.annotation.Configuration;
 
-@Configuration
-@ConfigurationProperties(prefix = TABLE_REPLICATION_TABLE_PARAMETERS)
-public class TableParametersConfig {
-
-  public static final String TABLE_REPLICATION_TABLE_PARAMETERS = "table-properties";
+import com.hotels.bdp.circustrain.api.conf.TransformOptions;
 
-  private Map<String, String> parameters;
+@Configuration(TRANSFORM_OPTIONS_BEAN)
+@ConfigurationProperties(TRANSFORM_OPTIONS_PROPERTY)
```

Comment:
Why does this no longer need the `prefix = ` bit? The build passes in Travis so it doesn't appear to matter, I just can't remember exactly how this works...

Reply:
I think the default argument is `value` and they're interchangeable. So you can either just put a string straight in and Spring will set prefix to `value`, or you can specify `prefix = ...`